### PR TITLE
add tinyMediaManager 2.8.1

### PR DESCRIPTION
**Pull request by ninjakttty**
#### Adding a new cask
- [x] Checked there aren’t open [pull requests](https://github.com/caskroom/homebrew-cask/pulls) for the same cask.
- [x] Checked there aren’t closed [issues](https://github.com/caskroom/homebrew-cask/issues) where that cask was already refused.
- [ ] When naming the cask, followed the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] Commit message includes cask’s name.
- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [ ] `brew cask install {{cask_file}}` worked successfully.
- [ ] `brew cask uninstall {{cask_file}}` worked successfully.

### CODE REVIEW

File: Casks/tinyMediaManager.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+cask 'tinyMediaManager' do
+  version '2.8.1'
+  sha256 '0b0f9e24372ace9cdfbfe5c5bce3b8fefa85f0ff2b71d01bc32b30aeb26186a6'
+
+  url 'http://release.tinymediamanager.org/dist/tmm_2.8.1_ced2842_mac.zip'
```
Comment (fanquake) [inline Casks/tinyMediaManager.rb]: You should use #{version} in the url stanza

**Comment by adidalal** [comment]
You need to read and follow the PR template, please.

File: Casks/tinyMediaManager.rb [hunk 2]
```diff
@@ -0,0 +1,15 @@
+cask 'tinyMediaManager' do
```
Comment (adidalal) [inline Casks/tinyMediaManager.rb]: should be `tinymediamanager` and filename changed accordingly

File: Casks/tinyMediaManager.rb [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+  url 'http://release.tinymediamanager.org/dist/tmm_2.8.1_ced2842_mac.zip'
+  name 'tinyMediaManager'
+  homepage 'http://www.tinymediamanager.org/'
+  license :apache
+
+  app 'tinyMediaManager.app'
+
+  caveats do
+    depends_on_java('6')
```
Comment (adidalal) [inline Casks/tinyMediaManager.rb]: Site says "You need at least Java 1.7 to run tinyMediaManager" - should be `depends_on_java('7+')`

**Comment by adidalal** [comment]
Thank you for the contribution. It needed some fixes, so they were made in 1dfe36bbfe58e1a67b84295444f8c35da23204a5. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

**Pull request closed without merging**